values.sample.yaml

```diff
@@ -88,6 +95,13 @@ tonicai:
       http: 80
   worker:
     #image: quay.io/tonicai/tonic_worker
+    #cpu:
```

Comment:
I'm guessing this is to make it easier for us to iterate on limits down the road if/when we need them? Otherwise, adding comments doesn't seem too useful.